modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/components/page-accessibility-tool/components/_Occurrence.scss

```diff
@@ -0,0 +1,17 @@
+@import 'atlas-variables';
```

Comment:
I think, you don't need to add the `_` prefix here. We usually use it when declaring in the css folder to import in the `main.scss`.

Reply:
These SCSS files are now imported as the pattern followed in frontend-js-a11y-web!